File: app/src/main/java/ru/vegax/xavier/a3test/user_data/UserAlbum.java
package ru.vegax.xavier.a3test.user_data;

public class UserAlbum{


    private final int mUserId;
    private final int mId;
    private final String mTitle;

    public UserAlbum(int userId, int id, String title) {
        mUserId = userId;
        mId = id;
        mTitle = title;
    }
    public int getUserId() {
        return mUserId;
    }

    public int getId() {
        return mId;
    }

    public String getTitle() {
        return mTitle;
    }
}

File: app/src/main/java/ru/vegax/xavier/a3test/models/UserAlbum.kt
package ru.vegax.xavier.a3test.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class UserAlbum {

    @SerializedName("userId")
    @Expose
    var userId: Int = 0
    @SerializedName("id")
    @Expose
    var id: Int = 0
    @SerializedName("title")
    @Expose
    var title: String? = null

}
